web/public/mgmt/config.yml

```diff
@@ -3424,7 +3432,7 @@ collections:
     sortable_fields: ["name", "filename"]
     create: true
     editor:
-      preview: true
+      preview: false
```

Comment:
AnytimeActionLInk no longer has a preview component since the removal of AnytimeActionTiles.